File: salesTracker/ClientApp/src/components/Sale.js
import React, { useState } from 'react';
import { Row, Col } from 'reactstrap';

const Sale = ({ id, details }) => {
    const [isOpen, setIsOpen] = useState(false)
    return (
        <tr xs="4">
            <td>{details.Product}</td>
            <td>{details.SalesPerson}</td>
            <td>{details.Customer}</td>
            <td>{details.Date}</td>
        </tr>
    )
}

export default Sale;

File: salesTracker/Entities/Sale.cs
using System;
namespace salesTracker.Entities
{
    public class Sale
    {
        public int SaleID { get; set; }
        public string Product { get; set; }
        public string SalesPerson { get; set; }
        public string Customer { get; set; }
        public DateTime Date { get; set; }
    }
}
